packages/@aws-cdk/aws-lambda/README.md

```diff
@@ -772,6 +772,19 @@ CodeGuru profiling is supported for all Java runtimes and Python3.6+ runtimes.
 See [the AWS documentation](https://docs.aws.amazon.com/codeguru/latest/profiler-ug/setting-up-lambda.html)
 to learn more about AWS Lambda's Profiling support.
 
+## Lambda with SnapStart
+
+```ts
+const fn = new lambda.Function(this, 'MyFunction', {
+  code: lambda.Code.fromAsset('handler.zip'),
+  runtime: lambda.Runtime.JAVA_11,
+  handler: 'example.Handler::handleRequest',
+  snapStart: true,
+});
+```
+
+See [the AWS documentation](https://docs.aws.amazon.com/lambda/latest/dg/snapstart.html) to learn more about AWS Lambda SnapStart
+
```

Comment:
AFAIK, Lambda SnapStart only works with published versions. If we just create a lambda function like this we only get $latest version, which I am afraid is not supported by SnapStart. Maybe we should create a `lambda.Version` for the function afterwards and address this notice in the readme?

Reply:
Also, SnapStart only supports JAVA_11 runtime atm. Worth a notice in the README or/and doc string in the `snapStart` prop. Thoughts?

Reply:
Overall, thanks for your contribution. I can't wait to see it being officially supported as L2 from the community contribution.

Reply:
Correct, snapStart is put into effect only on published versions. Using function.currentVersion as it's the recommended practice rather than managing lambda versions through lambda.Version

Added java 11 runtime information to both README and doc string above snap start